protobufs/livekit_ingress.proto

```diff
@@ -134,8 +134,9 @@ message IngressInfo {
   string participant_metadata = 14;
   bool reusable = 11;
   IngressState state = 12; // Description of error/stream non compliance and debug info for publisher otherwise (received bitrate, resolution, bandwidth)
+  optional bool session_enabled = 16;
```

Comment:
I would simply use `enabled` here, rather than `session_enabled` as we currently do not use a session semantic in this API. Please also add a comment specifying that when false, new connection attempts to this ingress will fail, and that the default value is `true`.